### PR TITLE
EL-8793 - Implemented individual asset progress update

### PR DESCRIPTION
**Pull request by PradheepKumarA**
* Implemented individual asset progress update

Doctor Service: https://github.com/cloud-elements/doctor-service/pull/13

### CODE REVIEW

File: src/core/vdrs/download/exportVdrs.js [hunk 1]
```diff
@@ -3,10 +3,29 @@
...
 
 
-module.exports = async (vdrNames) => {
+module.exports = async (vdrNames, service) => {
     let vdrs = {};
     for (var index = 0; index < vdrNames.length; index++) {
-        const vdr = await get(`/vdrs/${vdrNames[index]}/export`, "");
+        let vdr = null;
+        try {
+            if (service) {
```
Comment (GiriNellore) [inline src/core/vdrs/download/exportVdrs.js:11]: **question:** Would we ever have a situation of service being null? If not, how about removing this condition check in all the places?

File: src/util/createElements.js [hunk 2]
```diff
@@ -9,64 +9,82 @@ const update = require('./update');
...
+                        console.log(`Element Exteneded: ${element.key}`);
+                        if (exists > -1) {
+                            // 3. If resource already exist - PUT
+                            console.log('destinationResources[exists].id', destinationResources[exists].id);
+                            toUpload.update.push(update(`elements/${element.key}/resources/${destinationResources[exists].id}`, element.resources[i]));
+                            console.log(`    Resource Updated:\n        Method: ${element.resources[i].method}\n        Path: ${element.resources[i].path}`);
+                        } else {
+                            // 4. If resource doesn't exist - POST        
+                            toUpload.create.push(post(`elements/${element.key}/resources`, element.resources[i]));
+                            console.log(`    Resource Created:\n        Method: ${element.resources[i].method}\n        Path: ${element.resources[i].path}`);
```
Comment (GiriNellore) [inline src/util/createElements.js:53]: **suggestion:** Reduce the number of consoles as some of them are trivial.

File: src/core/vdrs/download/exportVdrs.js (hunk 1 above)
Comment (PradheepKumarA) [reply src/core/vdrs/download/exportVdrs.js:11]: This is the common code between cli and the doctor service. Service will be null for the cli

File: src/util/createElements.js (hunk 2 above)
Comment (PradheepKumarA) [reply src/util/createElements.js:53]: I haven't picked up the task to remove the console logs yet. Will keep that as a separate effort to clean up the whole project based on the cli/service condition.

**Pull request merged**